content/en/standard-attributes/_index.md

```diff
@@ -0,0 +1,1807 @@
+    description: The resource provider name. Default is `unknown`.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Resource (Browser events)
+  
+  - name: resource.provider.domain
+    description: The resource provider domain.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Resource (Browser events)
+  
+  - name: resource.provider.type
+    description: The resource provider type (for example, `first-party`, `cdn`, `ad`, or `analytics`).
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Resource (Browser events)
+    
+  - name: error.type
+    description: The error type (or error code in some cases).
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Source errors (Browser events)
+
+  - name: action.frustration.type:dead_click
+    description: The dead clicks detected by the RUM Browser SDK.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Frustration signals (Browser events)
+  
+  - name: action.frustration.type:rage_click
+    description: The rage clicks detected by the RUM Browser SDK.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Frustration signals (Browser events)
+  
+  - name: action.frustration.type:error_click
+    description: The error clicks detected by the RUM Browser SDK.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: Frustration signals (Browser events)
+  
+  - name: view.url_query.utm_source
+    description: The parameter in the URL tracking the source of traffic.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_medium
+    description: The parameter in the URL tracking the channel where the traffic is coming from.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_campaign
+    description: The parameter in the URL identifying the specific marketing campaign tied to that view.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_content
+    description: The parameter in the URL identifying the specific element a user clicked within a marketing campaign.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+  - name: view.url_query.utm_term
+    description: The parameter in the URL tracking the keyword a user searched to trigger a given campaign.
+    product_source:
+      - icon-rum
+      - browser
+    type: string
+    domain: UTM (Browser events)
+  
+
+
+  - name: language
+    description: The client SDK language used to generate the span. It can be one of cpp, dotnet, go, jvm, javascript, php, python, ruby.
```

Comment:
```suggestion
    description: The client SDK language used to generate the span. It can be one of `cpp`, `dotnet`, `go`, `jvm`, `javascript`, `php`, `python`, `ruby`.
```